monitor/dbt_project/macros/get_new_model_alerts.sql

```diff
@@ -9,6 +9,15 @@
     {% set new_alerts = [] %}
     {% for model_result_alert_dict in model_result_alert_dicts %}
         {% set status = elementary.insensitive_get_dict_value(model_result_alert_dict, 'status') | lower %}
+        {% set meta = fromjson(elementary.insensitive_get_dict_value(model_result_alert_dict, 'meta', '{}')) %}
```

Comment:
I think meta will be a string of a dumped json (see insert_dicts). I think you need to parse it and we already do it in the python so maybe worth extracting the subscribers and slack channel there? you can also wrap it with a try except to avoid failing the alert because of a parsing issue

Reply:
It will, this is why I use fromjson.
I am not sure why it is better to move the logic to the python outside of the macro.
Can you elaborate 🤔?